src/backends/pytorch/libtorch_backend.cc

```diff
@@ -30,6 +30,7 @@
 #include <exception>
 #include <memory>
 #include "src/core/constants.h"
+#include "src/core/cuda_utils.h"
```

Comment:
Do you need this include?

Reply:
Also removed from caffe2 backend as it was no longer necessary